kernel/kernel-impl/src/main/java/org/sakaiproject/util/impl/FormattedTextImpl.java

```diff
@@ -262,6 +264,11 @@ private Policy readPolicyFile(URL url) throws PolicyException {
     */
     public String removeSurrogates(String str) {
         StringBuilder sb = new StringBuilder();
+
+        if (EmojiManager.containsEmoji(str)) {
```

Comment:
Since this addresses that the replacement should use html entities then it should probably check the property
`content.cleaner.filter.utf8.replacement` and replace if it matches "html-entity", thoughts?